providers/song-info.js

```diff
@@ -57,8 +53,8 @@ const songInfo = {
 
 const handleData = async (responseText, win) => {
 	let data = JSON.parse(responseText);
-	songInfo.title = data.videoDetails?.media?.song || data?.videoDetails?.title;
-	songInfo.artist = data.videoDetails?.media?.artist || await getArtist(win) || cleanupArtistName(data?.videoDetails?.author);
```

Comment:
.videoDetails?.media? only appears in data parsed from ytpl.getInfo()
(useless here)